burn-book/src/import/pytorch-model.md

```diff
@@ -347,6 +347,58 @@ let record = PyTorchFileRecorder::<FullPrecisionSettings>::default()
     .expect("Should decode state successfully")
 ```
 
+### Models containing enum modules
+
+Burn supports models containing enum modules with new-type variants (tuple with one item). Importing
+weights for such models is supported by using the `PyTorchFileRecorder` automatically. However, it
```

Comment:
Minor syntax reword:

Importing weights for such models is automatically supported by the `PyTorchFileRecorder`. However, ...